tests/test_frame.py

```diff
@@ -2040,8 +2040,10 @@ def test_series_groupby_and_value_counts() -> None:
             "Max Speed": [380, 370, 24, 26],
         }
     )
-    c: pd.Series = df.groupby("Animal")["Max Speed"].value_counts()
-    check(assert_type(c, pd.Series), pd.Series)
+    c1 = df.groupby("Animal")["Max Speed"].value_counts()
+    c2 = df.groupby("Animal")["Max Speed"].value_counts(normalize=True)
+    check(assert_type(c1, pd.Series), pd.Series)
```

Comment:
This should be `check(assert_type(c1, "pd.Series[int]"), pd.Series, int)` and similar for the next one.

Reply:
done the changes, I have pushed the commit can you please check `poe pytest` fails at `test_pandas` at line 1443

Reply:
done the changes, I have pushed the commit can you please check `poe pytest` fails at `test_pandas` at line 1443